.github/ISSUE_TEMPLATE.md

```diff
@@ -1,30 +1,7 @@
-#### Issue description
-...
+<!--
+1. Check the version of package you are using. If it's not the newest version, update and try again (see changelog while updating!).
+2. If the issue is still there, write a minimal project showing the problem and expected output.
+3. Link to the project and mention Node version and OS in your report.
 
-#### Technical info
-* Webpack Bundle Analyzer version:
-* Webpack version:
-* Node.js version:
-* npm/yarn version:
-* OS:
-
-#### Debug info
-How do you use this module? As CLI utility or as plugin?
-
-If CLI, what command was used? (e.g. `webpack-bundle-analyzer -O path/to/stats.json`)
-
-If plugin, what options were provided? (e.g. `new BundleAnalyzerPlugin({ analyzerMode: 'disabled', generateStatsFile: true })`)
-
-What other Webpack plugins were used?
-
-It would be nice to also attach webpack stats file.
-It can be generated using these options:
-```js
-new BundleAnalyzerPlugin({
-  analyzerMode: 'disabled',
-  generateStatsFile: true,
-  // Excludes module sources from stats file so there won't be any sensitive data
-  statsOptions: { source: false }
-})`
-```
-`stats.json` will be created in Webpack bundle output directory.
+**IMPORTANT! You should use [Stack Overflow](https://stackoverflow.com/) for support related questions.**
+-->
```

Comment:
I'm not really sure what to do with these changes. We used to ask for the webpack stats file but I'm not sure if it's OK to go changing this issue template from what `webpack-defaults` generated.